tests/playwright/README.md

```diff
@@ -1,146 +1,103 @@
 
-```custom-extended-hook.ts
-import type { MyTestContext } from '../testContext/my-test-context';
-// or use one provided in @podman-desktop/tests-playwright
-import { afterEach } from 'vitest';
-import { takeScreenshotHook } from '@podman-desktop/tests-playwright';
+- Run `pnpm playwright init`, which will generate the `playwright.config` file, set up the testing folder structure and install playwright, or
+- Run `pnpm playwright install`, which will install the necessary binaries, and then run `playwright codegen --save-config=playwright.config.ts` which will generate the default `playwright.config` file, or
+- Add manually to your project the default `playwright.config` file:
 
-afterEach(async (context: MyTestContext) => {
-  context.onTestFailed(async () => await takeScreenshotHook(context.pdRunner, context.task.name));
+```
+import { defineConfig } from '@playwright/test';
+
+export default defineConfig({
+  testDir: './tests',
+  retries: 0,
+  use: {
+    headless: true,
+    viewport: { width: 1280, height: 720 },
+    ignoreHTTPSErrors: true,
+    screenshot: 'only-on-failure',
+    video: 'retain-on-failure',
+  },
 });
 ```
 
-### Global Setup file configuraiton
-
-Adding Global Setup/teardown module, class available in @podman-desktop/tests-playwright
-
-```global-setup.ts
-import { removeFolderIfExists } from '@podman-desktop/tests-playwright';
-
-let setupCalled = false;
-let teardownCalled = false;
-
-export async function setup() {
-  if (!setupCalled) {
-    // remove all previous testing output files
-    // Junit reporter output file is created before we can clean up output folders
-    // It is not possible to remove junit output file because it is opened by the process already, at least on windows
-    if (!process.env.CI) {
-      await removeFolderIfExists('tests/output');
-    } else {
-      console.log(
-        `On CI, skipping before All tests/output cleanup, see https://github.com/containers/podman-desktop/issues/5460`,
-      );
-    }
-    setupCalled = true;
-  }
-}
+3. Get into the Podman Desktop repo folder (should be named `podman-desktop`)
+4. Install its local dependencies by executing `pnpm install`
+5. Build the application's tests with `pnpm test:e2e:build`
+6. Go back into your repo folder and add the following script target to your package.json file, changing **'/path/to/podman-desktop'** by the actual absolute path of your `podman-desktop` repo folder:
 
-export async function teardown() {
-  if (!teardownCalled) {
-    // here comes teardown logic
-    teardownCalled = true;
+```
+  "scripts": {
+    "test:e2e": "PODMAN_DESKTOP_ARGS='/path/to/podman-desktop' playwright test tests/src/ --workers=1 --no-parallel",
   }
-}
 ```
 
-### Vitest Config
+7. Now you can implement your E2E tests in the `tests` folder of YOUR repo
+8. Execute `npm test:e2e` in order to run them
+9. When the tests finish you will find the test artifacts and logs available under `./tests/output`
```

Comment:
that would be a case, if `outputDir` property would be set in playwright config. Or is it default relative to `testDir` ?

Reply:
In playwright.config.ts, we have it defined currently like this: `outputDir: 'tests/playwright/output/'`
I've changed it for the next update